pkg/kubecfg/diff.go

```diff
@@ -193,19 +193,24 @@ func diff(desc, aName, bName, strategy string, aObj, bObj map[string]interface{}
 	return false, nil
 }
 
-func hash(discovery *discovery.DiscoveryInterface, obj *unstructured.Unstructured) string {
+// hash serves as an identifier for the Kubernetes resource.
+func hash(discovery *discovery.DiscoveryInterface, obj *unstructured.Unstructured, fqName bool) string {
```

Comment:
Does the `fqName` stuff belong in the commit `Remove client-go bindings from cmd`? It seems like it should just be removing the `client-go` flags.

Also, I think these commits are out of order?

Reply:
urgh, I think I did a bad rebase. Thanks for catching.